pkg/scsi/sbc.go

```diff
@@ -504,10 +504,14 @@ func SBCReadCapacity(host int, cmd *api.SCSICommand) api.SAMStat {
 	// data[1] = __cpu_to_be32(1U << bshift);
 	copy(cmd.InSDBBuffer.Buffer[4:], util.MarshalUint32(uint32(1<<bshift)))
 overflow:
-	cmd.InSDBBuffer.Resid = 8
+	if cmd.InSDBBuffer != nil {
+		cmd.InSDBBuffer.Resid = 0
```

Comment:
are we supposed to set to 8 in this case?

Reply:
by the time, code reaches here, InSDBBuffer was accessed.. so, this NULL is not required

Reply:
NULL check* is not required